finite_state_sdk/__init__.py

```diff
@@ -246,6 +274,11 @@ def create_asset_version(token, organization_context, business_unit_id=None, cre
         variables["input"]["ctx"]["products"] = product_id
 
     response = send_graphql_query(token, organization_context, graphql_query, variables)
+
+    if response.ok:
```

Comment:
Note: GraphQL return values can be HTTP "OK", but contain errors in the return value instead of "data". The `send_graphql_query` checks for this, and raises an exception with the errors if the response contains `errors` - I'll add a test for this, because the response is not a `requests` object, but the actual JSON that gets returned